src/main/java/com/example/ninthhomework/domain/user/service/CharactersService.java

```diff
@@ -1,5 +1,6 @@
 package com.example.ninthhomework.domain.user.service;
 
+import com.example.ninthhomework.Controller.CreateForm;
```

Comment:
Javaではパッケージ名は全て小文字で定義します。
```suggestion
import com.example.ninthhomework.controller.CreateForm;
```

Reply:
パッケージ名は小文字承知しました。
ad22f57d5e4dc9f6f0cb16ebe454a9b72435a0bb